.github/workflows/release.yml

```diff
@@ -108,49 +108,49 @@ jobs:
-        dotnet nuget push *.nupkg --source $env:USERPROFILE/.nuget/packages
-        dotnet new install IceRpc.Templates*.nupkg
-      if: runner.os == 'Windows'
-      shell: powershell
-    - name: Set version from tag
-      run: |
-        VERSION=${GITHUB_REF#refs/tags/v}
-        echo "VERSION=$VERSION" >> $GITHUB_ENV
-      if: runner.os == 'macOS' || runner.os == 'Linux'
-    - name: Set version from tag
-      run: |
-        $version = $env:GITHUB_REF -replace '^refs/tags/v',''
-        echo "VERSION=$version" >> $env:GITHUB_ENV
-      if: runner.os == 'Windows'
-    - name: 🔨 Build Examples
-      run: |
-        for solution in examples/*/*/*.sln; do dotnet build "$solution"; done
-      if: runner.os == 'macOS' || runner.os == 'Linux'
-      shell: bash
-    - name: 🔨 Build Examples
-      run: |
-        $examples = Get-ChildItem -Path examples -Recurse -Include *.sln
-        foreach ($example in $examples) { dotnet build $example.FullName }
-      if: runner.os == 'Windows'
-      shell: powershell
-    - name: 🧪 Test Templates
-      run: |
-        uses: ./.github/actions/test-templates
+      - uses: actions/checkout@v4
+      - name: Initialize NuGet local cache
+        run: dotnet restore
+      - name: Download packages artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: packages
+          path: packages
+      - name: Install NuGet packages and dotnet new templates
+        working-directory: packages
+        run: |
+          dotnet nuget push *.nupkg --source ~/.nuget/packages
+          dotnet new install IceRpc.Templates*.nupkg
+        if: runner.os == 'macOS' || runner.os == 'Linux'
+        shell: bash
+      - name: Install NuGet packages and dotnet new templates
+        working-directory: packages
+        run: |
+          dotnet nuget push *.nupkg --source $env:USERPROFILE/.nuget/packages
+          dotnet new install IceRpc.Templates*.nupkg
+        if: runner.os == 'Windows'
+        shell: powershell
+      - name: Set version from tag
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/v}
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+        if: runner.os == 'macOS' || runner.os == 'Linux'
+      - name: Set version from tag
+        run: |
+          $version = $env:GITHUB_REF -replace '^refs/tags/v',''
+          echo "VERSION=$version" >> $env:GITHUB_ENV
+        if: runner.os == 'Windows'
+      - name: 🔨 Build Examples
+        run: |
+          for solution in examples/*/*/*.sln; do dotnet build "$solution"; done
+        if: runner.os == 'macOS' || runner.os == 'Linux'
+        shell: bash
+      - name: 🔨 Build Examples
+        run: |
+          $examples = Get-ChildItem -Path examples -Recurse -Include *.sln
+          foreach ($example in $examples) { dotnet build $example.FullName }
+        if: runner.os == 'Windows'
+        shell: powershell
+      - name: 🧪 Test Templates
+        run: |
+          uses: ./.github/actions/test-templates
```

Comment:
```suggestion
      - name: 🧪 Test Templates
        uses: ./.github/actions/test-templates
```

Reply:
Just missed your suggestion, already fixed it in https://github.com/icerpc/icerpc-csharp/pull/3975/commits/cca77eaf8feba31f92035df81d2c9cb315e1df50